src/Apps/Checkout/Components/PaymentForm.tsx

```diff
@@ -0,0 +1,152 @@
+import { Elements, useElements, PaymentElement } from "@stripe/react-stripe-js"
+import {
+  type StripeElementsOptions,
+  type StripeElementsUpdateOptions,
+  type StripePaymentElementOptions,
+  loadStripe,
+} from "@stripe/stripe-js"
+import { Box, Text, Spacer } from "@artsy/palette"
+import { useState } from "react"
+import { FadeInBox } from "Components/FadeInBox"
+import { Collapse } from "Apps/Order/Components/Collapse"
+import { getENV } from "Utils/getENV"
+
+const stripePromise = loadStripe(getENV("STRIPE_PUBLISHABLE_KEY"))
+
+export const PaymentForm: React.FC = () => {
+  const orderOptions: StripeElementsUpdateOptions = {
+    amount: 12345,
+    currency: "eur",
+    // onBehalfOf: "acct_14FIYS4fSw9JrcJy", // US
+    onBehalfOf: "acct_1KMqw0A6DHSUKBik", // DE
```

Comment:
This is hardcoded on purpose. This data will come from the Order in the future.
Me and @starsirius  discussed an option to get the amount and currency from the artwork via query params, but this logic will be discarded in the future, so I am still considering it.

Reply:
From what I understood, we will remove it in the future since the data will come from the order, can we maybe add a comment so we don't forget to remove it?